netbox/dcim/api/serializers.py

```diff
@@ -171,6 +171,16 @@ class Meta:
         fields = ['id', 'rack', 'units', 'created', 'user', 'tenant', 'description']
 
 
+class RackElevationDetailFilterSerializer(serializers.Serializer):
+    face = serializers.ChoiceField(choices=['front', 'rear'], default='front')
+    render_format = serializers.ChoiceField(choices=['json', 'svg'], default='json')
+    width = serializers.IntegerField(default=230)
```

Comment:
This conflicts with `RackFilter.width`. As I mentioned earlier it would be good to use `unit_width` and `unit_height` to make obvious the relationship between these two fields.